script.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Check if `git` exists in the current env.
+if ! command -v git &> /dev/null; then
+    echo "git could not be found"
+    exit 2
+fi
```

Comment:
This check is useless here and now, the action is run in a GitHub Runner, it has `git` available. If not, with `set -e`  it will be captured when trying to execute `git log`.